application_sdk/outputs/parquet.py

```diff
@@ -72,6 +122,15 @@ def __init__(
             self.output_path = os.path.join(self.output_path, self.typename)
         os.makedirs(self.output_path, exist_ok=True)
 
+        # For Query Extraction
+        if self.start_marker and self.end_marker:
+            self.path_gen = (
+                lambda chunk_start,
+                chunk_count,
+                start_marker,
+                end_marker: f"{self.start_marker}_{self.end_marker}.parquet"
+            )
```

Comment:
did not get why this is needed, considering all method that requires file_path, generates one using the `path_gen` function